lib/qubicInterface.ts

```diff
@@ -70,6 +72,60 @@ export class QubicInterface {
     };
   }
 
+  //Signs a raw arraybuffer with a seed
```

Comment:
@ahmed-tarek-salem I think we can these 3 methods reuse most of the code and include the prepend message required here https://github.com/qubic/wallet-app/issues/235.
Would you please have a look?

Reply:
Agreed will be adjusted later by @ahmed-tarek-salem